doc/requirements.txt

```diff
@@ -33,4 +33,4 @@ lxml_html_clean
 --extra-index-url https://test.pypi.org/simple/
 pennylane-lightning-kokkos==0.43.0.dev12
 pennylane-lightning==0.43.0.dev12
-pennylane==0.43.0.dev24
+pennylane==0.43.0.dev30
```

Comment:
@paul0403 Actually, if one does not change this file syncly, the doc CI runner will fail immediately basically

Reply:
Yes